java-api-core/src/com/kinvey/java/query/AbstractQuery.java

```diff
@@ -111,6 +116,140 @@ public String getQueryFilterJson(JsonFactory factory) {
         return jsonResult;
     }
 
+    private void buildQueryString(JsonGenerator generator, AbstractMap<String, Object> filterMap) throws IOException {
```

Comment:
How does this relate to `pullBlocking`?

Reply:
Interesting.  Actually this does not relate to `pullBlocking`, it is part of another [PR](https://github.com/Kinvey/java-library/pull/104) for querying. I may have accidentally branched this off of the query branch, which is why it is showing up in this PR. If it's ok, I'll merge the other PR, which has already been approved, prior to merging this when its ready. That way, there will be no conflicts.

Reply:
:+1: